diesel_derives/src/lib.rs

```diff
@@ -18,6 +18,8 @@
     clippy::items_after_statements,
     clippy::used_underscore_binding
 )]
+// Nightly Regression https://github.com/rust-lang/rust/issues/70814
+#![cfg_attr(feature = "nightly", allow(unused_braces))]
```

Comment:
If you want to allow this lint only in nightly CI, you can pass the flag on CI config instead of adding new feature.